### PR TITLE
Improve efficiency of stats

### PR DESCRIPTION
**Pull request by bunsenmcdubbs**
Back down to ~2 sec

### CODE REVIEW

File: server/branch.ts [hunk 1]
```diff
@@ -42,20 +44,23 @@ export class NoopBranch {
 	}
 
 	public async loadFromSchema(): Promise<this> {
-		let questionBranches: QuestionBranches = JSON.parse(await readFileAsync(this.location));
-		let schema = JSON.parse(await readFileAsync(path.resolve(__dirname, SCHEMA_FILE)));
+		let questionBranches = QuestionBranchCache;
+		if (!questionBranches) {
+			questionBranches = QuestionBranchCache = JSON.parse(await readFileAsync(this.location));
```
Comment (illegalprime) [inline server/branch.ts]: assignment on separate lines pls.

File: server/routes/templates.ts [hunk 2]
```diff
@@ -531,13 +531,19 @@ templateRoutes.route("/admin").get(authenticateWithRedirect, async (request, res
 			maxTeamSize: config.maxTeamSize.toString()
 		}
 	};
+
+	interface IApplicationMap {
+			[key: string]: Branches.ApplicationBranch;
```
Comment (illegalprime) [inline server/routes/templates.ts]: why double tabs here?

File: server/branch.ts (hunk 1 above)
Comment (illegalprime) [inline server/branch.ts]: shouldn't this also be cached per `this.location`?

File: server/routes/templates.ts (hunk 2 above)
Comment (petschekr) [inline server/routes/templates.ts]: 2 tabs here :(

File: server/branch.ts [hunk 3]
```diff
@@ -18,7 +18,10 @@ type QuestionBranch = QuestionBranchTypes[keyof QuestionBranchTypes];
 
 const SCHEMA_FILE: string = "./config/questions.schema.json";
 
-let QuestionBranchCache: QuestionBranches;
+interface IQuestionBranchCache {
+		[location: string]: QuestionBranches;
```
Comment (illegalprime) [inline server/branch.ts]: double tab

**Comment by petschekr** [comment]
Version should be bumped to 1.10.1 before being merged

**Pull request merged**